packages/fxa-content-server/app/scripts/lib/sentry.js

```diff
@@ -4,7 +4,7 @@
 
 import _ from 'underscore';
 import Logger from './logger';
-import Raven from 'raven';
+import * as Sentry from '@sentry/browser';
```

Comment:
Not possible to skip the `* as Sentry` here and just use `Sentry`?

Reply:
This is because of the way they have packaged `@sentry/browser`. There isn't a function named `Sentry` only a bunch of methods. So we need to do `* as Sentry` to get the Object to call the methods from. It's also recommended in their docs

Reply:
It's the same thing with Sentry in TypeScript as well, it always needs the `*` import.